README.md

```diff
@@ -117,6 +124,8 @@ require('core-js/build')({
   // ...
 });
 ```
+If your webpack build includes something like UglifyJs, then the excluded parts will be entirely stripped from the resulting code using dead code elimination.
```

Comment:
If you will exclude `something` - `require('something')` just will not be added :)